src/components/common/Dialogs/ControlSafeDialog/TransactionTypesSection/TransferFundsSection.tsx

```diff
@@ -109,22 +110,24 @@ const TransferFundsSection = ({
       );
       if (response.status === 200) {
         const data = (await response.json()) as SafeBalanceApiData[];
-        const formattedSafeBalances: SafeBalance[] = data.map(
-          (balanceData) => ({
+
+        const tokenFromDbPromises = data.map(async (balanceData) => {
+          const network = getNetworkFromChainName(chainName);
+          const token = await getTokenFromEveryWhereQuery(
+            balanceData?.tokenAddress || ADDRESS_ZERO,
+            network,
+          );
```

Comment:
Why this approach instead of the old one? The token data is already available in `balanceData`.

Reply:
My logic in making this change was that at the moment no tokens have their thumbnail string filled out, so in order to correctly set the thumbnail to show to the user in this step we need to fetch it from the database (as the `logoUri` returned via the safe API will in all likelihood not be the same as the thumbnail that we choose to store).

However, on coming back to this today I have reverted these changes as it does not seem worth the extra fetch to the database and this will be a problem to tackle when we figure out what thumbnails we are using for tokens and how we are storing them (I'm hoping that there will be a quicker way to get the thumbnail than just querying the database, but I may be wrong)

This is also why the DAI token will no longer show, as on second thought I believe it might be confusing if we set token thumbnails to tokens **only** when they have been fetched from a safe